spark-cassandra-connector/src/test/scala/com/datastax/spark/connector/cql/CassandraConnectorConfSpec.scala

```diff
@@ -53,30 +53,31 @@ class CassandraConnectorConfSpec extends FlatSpec with Matchers {
     connConf.cassandraSSLConf.enabledAlgorithms should contain theSameElementsAs Seq("TLS_ECDH_RSA_WITH_AES_256_CBC_SHA384", "TLS_DHE_RSA_WITH_AES_256_CBC_SHA256")
   }
 
-  it should "resolve default retry delay settings correctly" in {
+  // retry delay temporary disabled due to SPARKC-360
+  ignore should "resolve default retry delay settings correctly" in {
```

Comment:
Are those tests retained until Java driver implementation is changed so that we may reimplement delayed retries?